src/main/java/org/folio/edge/core/EdgeVerticle.java

```diff
@@ -51,7 +51,7 @@
   public final int port;
   public final String okapiURL;
   public final String apiKeySources;
-  public final long reqTimeoutMs;
+  public final int reqTimeoutMs;
```

Comment:
https://vertx.io/docs/apidocs/io/vertx/core/http/HttpClientRequest.html#setTimeout-long- takes a long, OkapiClient takes a long, why do you change it to int?